src/libs/Navigation/Navigation.ts

```diff
@@ -224,16 +227,40 @@ function goBack(fallbackRoute?: Route, shouldEnforceFallback = false, shouldPopT
     const isCentralPaneFocused = findFocusedRoute(navigationRef.current.getState())?.name === NAVIGATORS.CENTRAL_PANE_NAVIGATOR;
     const distanceFromPathInRootNavigator = getDistanceFromPathInRootNavigator(fallbackRoute ?? '');
 
-    // Allow CentralPane to use UP with fallback route if the path is not found in root navigator.
-    if (isCentralPaneFocused && fallbackRoute && distanceFromPathInRootNavigator === -1) {
-        navigate(fallbackRoute, CONST.NAVIGATION.TYPE.UP);
-        return;
+    if (isCentralPaneFocused && fallbackRoute) {
```

Comment:
This change is cosmetic only. It won't change behaviour. I just realised that the `ifs` are similar.